src/Test.UnitTests.BinSkim.Rules/BuildWithSecureToolsTests.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+using FluentAssertions;
+
+using Microsoft.CodeAnalysis.BinaryParsers.ProgramDatabase;
+using Microsoft.CodeAnalysis.IL.Sdk;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.IL.Rules
+{
+    public class BuildWithSecureToolsTests
+    {
+        private static readonly Random s_random;
+        private static readonly int s_randomSeed;
+
+        static BuildWithSecureToolsTests()
+        {
+            s_randomSeed = (int)DateTime.UtcNow.TimeOfDay.TotalMilliseconds;
+            s_random = new Random(s_randomSeed);
```

Comment:
>s_randomSeed [](http://example.com/codeflow?start=34&length=12)

You always need to report the random seed when you use this pattern.

Reply:
removed this in favor of a simple reversed list